### PR TITLE
[terra-menu] Added outline styles for menu header back button

### PR DESCRIPTION
**Pull request by PK106552**
### Summary

**What was changed:**
Added outline style to fix focus not going to sub menu header back button.

**Why it was changed:**
Focus is not going to sub menu header back button while using safari + keyboard navigation with arrow.

### Testing

This change was tested using:

- [ ] WDIO
- [ ] Jest
- [x] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [ ] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [ ] Accessibility review
- [ ] Functional review

### Additional Details

**This PR resolves:**

UXPLATFORM-9653

---

Thank you for contributing to Terra.
@cerner/terra

Screenshot 2023-09-21 at 4 35 49 PM

### CODE REVIEW

**Review by sdadn — APPROVED** [review]

**Comment by supreethmr** [comment]
what was the reason to create new screenshots for collapsible menu view in this PR. And also I am not able to see the screenshot with focus change made on this PR can you point me to that test which covers new behavior that is being added with this PR

**Comment by PK106552** [comment]
> what was the reason to create new screenshots for collapsible menu view in this PR. And also I am not able to see the screenshot with focus change made on this PR can you point me to that test which covers new behavior that is being added with this PR

Build was failed due to screenshots mismatch that's why i have updated new screenshots, and already test cases has been written for focus check  in submenu heading back button after pressing shift+tab key in menu-spec.js file.

**Comment by supreethmr** [comment]
> > what was the reason to create new screenshots for collapsible menu view in this PR. And also I am not able to see the screenshot with focus change made on this PR can you point me to that test which covers new behavior that is being added with this PR
> 
> Build was failed due to screenshots mismatch that's why i have updated new screenshots, and already test cases has been written for focus check in submenu heading back button after pressing shift+tab key in menu-spec.js file.

I think those screenshot updates are not required for this PR can check again. collapsible-menu-view screenshots are not modified one they are new screenshots so need to verify again since there are new tests added for collapsible Menu view in this PR these screenshots seems to be inappropriate.

**Comment by PK106552** [comment]
> > > what was the reason to create new screenshots for collapsible menu view in this PR. And also I am not able to see the screenshot with focus change made on this PR can you point me to that test which covers new behavior that is being added with this PR
> > 
> > 
> > Build was failed due to screenshots mismatch that's why i have updated new screenshots, and already test cases has been written for focus check in submenu heading back button after pressing shift+tab key in menu-spec.js file.
> 
> I think those screenshot updates are not required for this PR can check again. collapsible-menu-view screenshots are not modified one they are new screenshots so need to verify again since there are new tests added for collapsible Menu view in this PR these screenshots seems to be inappropriate.

Collapsible-menu-view test has been added 4 weeks ago by Aaron Johnson and that time screenshots has been added only for terra-default-theme but i don't know how they were missed to update remaining two themes, but when i run that spec file now those remaining two themes new screenshots has been updated in this PR and i have attached his PR for reference.
https://github.com/cerner/terra-framework/pull/1750

**Review by sugan2416 — APPROVED** [review]

**Comment by supreethmr** [comment]
> > > > what was the reason to create new screenshots for collapsible menu view in this PR. And also I am not able to see the screenshot with focus change made on this PR can you point me to that test which covers new behavior that is being added with this PR
> > > 
> > > 
> > > Build was failed due to screenshots mismatch that's why i have updated new screenshots, and already test cases has been written for focus check in submenu heading back button after pressing shift+tab key in menu-spec.js file.
> > 
> > 
> > I think those screenshot updates are not required for this PR can check again. collapsible-menu-view screenshots are not modified one they are new screenshots so need to verify again since there are new tests added for collapsible Menu view in this PR these screenshots seems to be inappropriate.
> 
> Collapsible-menu-view test has been added 4 weeks ago by Aaron Johnson and that time screenshots has been added only for terra-default-theme but i don't know how they were missed to update remaining two themes, but when i run that spec file now those remaining two themes new screenshots has been updated in this PR and i have attached his PR for reference. #1750

I Still believe those screenshots should go as different PR.

**Review by supreethmr — CHANGES_REQUESTED** [review]
~~ Not able figure out the changes made by this PR. It looks the focus style is same as before as shown in below screenshots. Is there any specific scenario where you don't see the focus highlight on back button. 
screenshot of Deployment Link ~~

**Comment by supreethmr** [comment]
> Not able figure out the changes made by this PR. It looks the focus style is same as before as shown in below screenshots. Is there any specific scenario where you don't see the focus highlight on back button. screenshot of Deployment Link :
> 
> Screenshot 2023-09-28 at 8 39 02 PM
> screenshot terra-ui site :
> 
> Screenshot 2023-09-28 at 8 39 12 PM

Got it !! Focus was not visible only on Safari Browser.

**Comment by supreethmr** [comment]
> ~Not able figure out the changes made by this PR. It looks the focus style is same as before as shown in below screenshots. Is there any specific scenario where you don't see the focus highlight on back button. screenshot of Deployment Link~

Got it !! this issue was happening only on Safari.

**Review by sdadn — APPROVED** [review]

**Comment by PK106552** [comment]
> > > > > what was the reason to create new screenshots for collapsible menu view in this PR. And also I am not able to see the screenshot with focus change made on this PR can you point me to that test which covers new behavior that is being added with this PR
> > > > 
> > > > 
> > > > Build was failed due to screenshots mismatch that's why i have updated new screenshots, and already test cases has been written for focus check in submenu heading back button after pressing shift+tab key in menu-spec.js file.
> > > 
> > > 
> > > I think those screenshot updates are not required for this PR can check again. collapsible-menu-view screenshots are not modified one they are new screenshots so need to verify again since there are new tests added for collapsible Menu view in this PR these screenshots seems to be inappropriate.
> > 
> > 
> > Collapsible-menu-view test has been added 4 weeks ago by Aaron Johnson and that time screenshots has been added only for terra-default-theme but i don't know how they were missed to update remaining two themes, but when i run that spec file now those remaining two themes new screenshots has been updated in this PR and i have attached his PR for reference. #1750
> 
> I Still believe those screenshots should go as different PR.

@supreethmr Removed newly added screenshots

File: packages/terra-menu/src/orion-fusion-theme/Menu.module.scss [hunk 1]
```diff
@@ -11,6 +11,7 @@
     --terra-menu-submenu-header-background-color: #f4f4f4;
     --terra-menu-submenu-header-border-bottom: 2px solid #9b9fa1;
     --terra-menu-submenu-header-button-outline-offset: -4px;
+    --terra-menu-submenu-header-button-outline: #004c76 solid 2px;
```
Comment (supreethmr) [inline packages/terra-menu/src/orion-fusion-theme/Menu.module.scss:14]: this variable is not present in Menu.module.scss and MenuItem.module.scss. why do we on theme files ..?

Comment (PK106552) [reply packages/terra-menu/src/orion-fusion-theme/Menu.module.scss]: Updated style file - https://github.com/cerner/terra-framework/pull/1801/commits/33e14a6220d3246fca3812a66d8090645a1b56f9

**Comment by supreethmr** [comment]
https://github.com/cerner/terra-framework/pull/1819

**Comment by supreethmr** [comment]
Closing this PR has Mulitple Screenshots were updated due to merge conflict. Created new PR : https://github.com/cerner/terra-framework/pull/1819 to address this issue

**Pull request closed without merging**